Assets/Scripts/ManagerScripts/CountdownManager.cs

```diff
@@ -50,9 +48,8 @@ private void Update()
                 countdownSound.Stop();
                 countdownUI.SetActive(false);
 
-                if (!_hasSetPlayerInputToTrue)
+                if (countingDown)
```

Comment:
This technically shouldn't be here because there's a bug in the if-else logic. I wrote a big comment about it in Judy's countdown PR (all these if-else statements are triggered every frame)

Reply:
is it something that will be addressed in her PR?

Reply:
that's why I had the state variable before, but using countdown as a state variable worked.

Reply:
Hopefully idk
This class is getting messy

Reply:
at least no extras from this PR, just using whats already there